packages/core/contracts/insured-bridge/implementation/L1_BridgeRouter.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.7.6;
+
+import "@eth-optimism/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol";
+
+import "./L2_BridgeDepositBox.sol";
+
+contract BridgeRouter is OVM_CrossDomainEnabled {
+    address public l1Owner;
+    address public l2DepositBox;
+
+    event AddedL2DepositContract(address l2Contract);
+    event AddedL2DepositContract(address originToken, address destinationToken);
```

Comment:
Is this event misnamed? Same as above

Reply:
good catch. was meant to be `WhitelistedToken`